docs/compression_algorithms/Sparsity.md

```diff
@@ -77,8 +77,8 @@ The magnitude sparsity method implements a naive approach that is based on the a
     "algorithm": "magnitude_sparsity",
     "initializer": {
         "batchnorm_adaptation": {
-            "num_bn_adaptation_steps": 10, // Number of batches from the training dataset to pass through the model at initialization in order to update batchnorm statistics of the original model
-            "num_bn_forget_steps": 5, // Number of batches from the training dataset to pass through the model at initialization in order to erase batchnorm statistics of the original model (using large momentum value for rolling mean updates)
+            "num_bn_adaptation_samples": 10, // Number of samples from the training dataset to pass through the model at initialization in order to update batchnorm statistics of the original model
+            "num_bn_forget_samples": 5, // Number of samples from the training dataset to pass through the model at initialization in order to erase batchnorm statistics of the original model (using large momentum value for rolling mean updates)
```

Comment:
Please update the values from 5/10 to the corresponding number of samples

Reply:
Done.